actors/test/actor-tests/src/test/java/cloud/orbit/actors/test/DeactivationTest.java

```diff
@@ -63,6 +63,7 @@ public void simpleCleanupTest() throws ExecutionException, InterruptedException
         final UUID id = actor1.getUniqueActivationId().join();
         clock.incrementTime(20, TimeUnit.MINUTES);
         stage.cleanup().join();
+        waitForDeactivations();
```

Comment:
Cleanup drops through now and various tests failed. For simplicity anywhere stage.cleanup is used I added a waitForDeactivations as that gives the same behavior for the tests.
In a real Orbit application we never rely on cleanup doing something immediately in pulse.